src/utils/image.ts

```diff
@@ -15,7 +15,7 @@ export function preprocessImage(
   if (imageData.width <= MAX_WIDTH && imageData.height <= MAX_WIDTH) return imageData
   const scale = MAX_WIDTH / Math.max(imageData.width, imageData.height)
   const [w, h] = [imageData.width * scale, imageData.height * scale]
-  const scaled = getScaled(w, h)
+  const scaled = getScaled(Math.round(w), Math.round(h))
```

Comment:
does it mean `w` or `h` may have a decimal part, and is there a reason to choose `round` rather than `floor` or `ceil`? (I'm not the developer of this library so just curious.)

cc @guanbinrui you may want review this?

Reply:
@Jack-Works that is correct. Only reason to opt for `round` is that it will find the nearest pixel. I.e 10.9px is probably better rounded to 11 than 10 to reduce distortion (though it isn't likely to be perceptible anyway).

Reply:
I'll wait for the response from the original developer, if there is none, I'll merge and release this next week. cc @guanbinrui again